kimchi/benches/proof_criterion.rs

```diff
@@ -22,7 +22,7 @@ pub fn bench_proof_creation(c: &mut Criterion) {
     group.sample_size(100).sampling_mode(SamplingMode::Auto);
     group.bench_function(
         format!("proof verification (SRS size 2^{})", ctx.srs_size()),
-        |b| b.iter(|| ctx.batch_verification(black_box(vec![proof_and_public.clone()]))),
+        |b| b.iter(|| ctx.batch_verification(black_box(&vec![proof_and_public.clone()]))),
```

Comment:
nit: you can just write `&[proof_and_public.clone()]` (I'm wondering why clippy doesn't say anything there again)

Reply:
It's much more aggressive in Rust v1.67.0 (see #970)